### PR TITLE
Fixing various things

### PR DESCRIPTION
**Pull request by Pauan**
In order to run the code, you must:

1. Run `wasm-pack build --scope fuyangl --release --target no-modules`

2. Run `node generate-sql.js`

3. Add the `base62.sql` file into BigQuery.

### CODE REVIEW

**Review by liufuyang — APPROVED** [review]
Thank you very much @Pauan  I will just merge first then try it out

File: generate-sql.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+const bytes = Array.from(new Uint8Array(buffer.buffer));
+
+fs.writeFileSync("base62.sql", `\
+CREATE TEMP FUNCTION \`b62_to_hex\`(b62 STRING) RETURNS STRING LANGUAGE js AS '''
+${glue}
+const bytes = new Uint8Array(${JSON.stringify(bytes)});
+
+return wasm_bindgen(bytes).then(() => {
+    return wasm_bindgen.run(b62);
+});
```
Comment (liufuyang) [inline generate-sql.js]: How come this would work? How do I tell this would call the `get_hex` or `get_b62` function?

Comment (Pauan) [reply generate-sql.js]: I had mistakenly thought you were only trying to call one function. I've updated it to export all the functions again.

**Comment by liufuyang** [comment]
Thank you again, this work pretty well :)

**Pull request merged**